Android/Fundamentals/drawables-ii/create-a-transition.md

```diff
@@ -0,0 +1,164 @@
+# create-a-transition
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: how to
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
```

Comment:
what's a transition?
what are the steps first?
List the code examples after, try to remove some of the boilerplate xml.